src/exporthtml.ts

```diff
@@ -10,22 +10,27 @@ import { minify }        from 'html-minifier';
 import { internalGenerateOptions, ObjectType, ReturnTypes } from './types';
 const template = fs.readFileSync(path.join(__dirname, 'template.html'), 'utf8');
 
-let options: Intl.DateTimeFormatOptions = {
+let optionsShort: Intl.DateTimeFormatOptions = {
+	day: "2-digit",
+    month: "2-digit",
+    year: "2-digit",
+};
+let optionsLong: Intl.DateTimeFormatOptions = {
 	weekday: 'long',
 	year: 'numeric',
-	month: 'short',
+	month: 'long',
 	day: 'numeric',
 	hour: '2-digit',
 	minute: '2-digit',
 };
```

Comment:
move to ./src/static.ts and export it as a const then import in exporthtml.ts so its a little cleaner

Reply:
looks like some of the lines i highlighted didnt appear here, what I'm referring to is the optionsShort, optionsLong, and date variables

Reply:
👍🏻

Reply:
Resolved.